### PR TITLE
Pass os.Environ() as environment to process from init.

### PR DESCRIPTION
**Pull request by LK4D4**
Replacement of #418

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: integration/execin_test.go [hunk 1]
```diff
@@ -245,3 +245,72 @@ func TestExecInTTY(t *testing.T) {
...
+	}
+	err = container.Start(process)
+	stdinR.Close()
+	defer stdinW.Close()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	buffers := newStdBuffers()
+	ps := &libcontainer.Process{
```
Comment (mrunalp) [inline integration/execin_test.go]: want to call this something else, maybe process2?

**Comment by mrunalp** [comment]
Minor comment; LGTM, otherwise.

**Comment by LK4D4** [comment]
@mrunalp Fixed

**Comment by mrunalp** [comment]
Thanks!

**Pull request merged**